src/utils/getPath.ts

```
@@ -1,12 +1,53 @@
+ * @param filePath - the blog post full file location
+ * @param includeBase - whether to include `/posts` in return value
+ * @returns blog post path
+ */
+export function getPath(
+  id: string,
+  filePath: string | undefined,
+  includeBase = true,
+  url?: string
+) {
+  if (url) {
+    console.log("getPath", url);
```

ESLint

[error] Unexpected console statement.

(no-console)